src/main/java/com/theelixrlabs/healthcare/model/Patient.java

```diff
@@ -0,0 +1,35 @@
+package com.theelixrlabs.healthcare.model;
+
+import com.theelixrlabs.healthcare.constants.PatientConstants;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.UUID;
+
+/**
+ * Entity class of Patient
+ */
+@AllArgsConstructor
+@Data
+@Document(collection = "patient")
```

Comment:
Is the collection name specific to single patient ?
if not update it to "patients"

Reply:
Resolved

Reply:
resolved